src/stories/StyledCoinListItem.tsx

```diff
@@ -112,7 +113,17 @@ export default function StyledCoinListItem({
                 )}`}</span>
                 <StyledIconButton icon={IconVariant.COPY} onClick={() => copy(asset.chainId)} isLoading={isCopying} />
                 {asset.chainId && onAdd && (
-                  <StyledIconButton icon={IconVariant.METAMASK_LOGO} onClick={() => onAdd(asset.chainId ?? '')} />
+                  <StyledIconButton
+                    icon={IconVariant.METAMASK_LOGO}
+                    onClick={() =>
+                      onAdd(
+                        asset.chainId ?? '',
+                        renderToString(
+                          <DfxAssetIcon asset={asset.name as AssetIconVariant} disabled={asset.comingSoon} />,
```

Comment:
I wouldn't set the disabled prop ... this would add the token with the greyish icon, if it's disabled 😁